kettle-plugins/hadoop-cluster/ui/pom.xml

```diff
@@ -68,6 +68,10 @@
       <groupId>pentaho</groupId>
       <artifactId>pentaho-i18n-webservice-bundle</artifactId>
     </dependency>
+    <dependency>
+      <groupId>pentaho</groupId>
+      <artifactId>pentaho-big-data-impl-clusterTests</artifactId>
```

Comment:
should this be test scope

Reply:
I think this one is inherited from its parent....